### ISSUE TITLE
Monsters deal 150% damage with 4 runes at nightmare

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#580]
See title

**Comment by jeremygurr** [comment]
ok try it now

**Comment by Sandman25DCSS** [comment]
Not sure if it works, it is less than 150% now.

> You shoot an arrow. The arrow closely misses the spriggan rider.
> _The spriggan rider hits you (14) from afar with a spear!
>  Damage: -11%(-19hp)   hp: 89%(151hp)
>  You shoot an arrow.

**Comment by Sandman25DCSS** [comment]
> The dire elephant tramples you (10)! You hold your ground!
> _The dire elephant trunk-slaps you but does no damage.
>  Damage: -7%(-12hp)   hp: 85%(146hp)

**Comment by Sandman25DCSS** [comment]
> The stone arrow hits you! (34)
> _Damage (40) was greater than 30% of your hp (131)!!!
>  Huge Dmg: -26%(-47hp) hp: 74%(131hp)

**Comment by jeremygurr** [comment]
Oh that is a different issue than what I thought you were talking about. Monsters were actually doing too much damage (not just a display issue), so I reduced the rune curse damage.

But your issue here is a display problem. Yes I did fix the first case you showed me of a "zapping" action. But these cases you are showing now are two more. I'll check those out. I'm going to have to come up with a more generic solution for this.
